RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronFinalizer_cfi.py

```diff
@@ -0,0 +1,45 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoEgamma.EgammaTools.regressionModifier_cfi import regressionModifier106XUL
+
+lowPtRegressionModifier = regressionModifier106XUL.clone(
```

Comment:
```suggestion
_lowPtRegressionModifier = regressionModifier106XUL.clone(
```
if there is no need to export this generically outside of this file

Reply:
Done.